src/components/navigation/index.js

```diff
@@ -16,6 +16,8 @@ function Navigation() {
     {label: 'Profile', icon: 'fa-user', path: '/profile'},
     {label: 'More', icon: 'fa-minus-circle', path: '/more'},
     {label: 'Login', icon: 'fa-user', path: '/login'},
+      {label: 'Signup', icon: 'fa-user', path: '/signup'},
```

Comment:
Looks like we are using the same icon for profile, login and signup. Maybe we can combine login and signup screens? Let's discuss this over a call